MpvRemote/app/src/main/java/miccah/mpvremote/MainActivity.java

```diff
@@ -229,6 +229,9 @@ public void playPauseButton(View view) {
         BackgroundImageButton button = (BackgroundImageButton)view;
         button.sendCommand("pause", "state", !button.getState());
     }
+    public void stopButton(View view) {
+        sendCommand(null, "stop");
```

Comment:
I need to look deeper, but I *think* this "stop" command is used in the context of fast forward / rewind to allow for the press and hold style of button.

When the user first presses the button, it'll send the "repeat" command to the server. When the user releases the button, it sends the "stop" command to the server.

Reply:
I double-checked this and indeed it calls self.controller.stop() which sends a "stop" command to mpv.

Regarding the button icon, there was no stop button in the collection so I downloaed one from the same family